GetUrl.cs

```diff
@@ -123,7 +116,7 @@ private async Task<string> GetUrlFromBackend(string bcchCommand)
                 RedirectStandardOutput = true,
                 RedirectStandardError = true
             };
-            var process = Process.Start(startInfo);
+            var process = Process.Start(startInfo) ?? throw new Exception("Failed to start the pwsh process!");

```

Comment:
```suggestion
            using var process = Process.Start(startInfo) ?? throw new Exception("Failed to start the pwsh process!");
```

If I'm not mistaking it would be better to explicit end the process (Dispose) by adding `using` statement

https://stackoverflow.com/questions/27300821/what-happens-to-a-process-in-a-using-statement-without-waitforexit